geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/RebalanceCommand.java

```diff
@@ -170,6 +170,13 @@ private void toCompositeResultData(ResultModel result,
     resultStr.append(CliStrings.REBALANCE__MSG__TOTALTIME).append(" = ").append(rstlist.get(8))
         .append(newLine);
 
+    table1.accumulate("Rebalanced Stats", CliStrings.REBALANCE__MSG__MEMBER_COUNT);
+    table1.accumulate("Value", String.valueOf(cache.getMembers().size()));
```

Comment:
@mkevo: this is what I was talking about, the output will contain the number of members _currently_ present in the distributed system, **_not_** the amount of members on which the rebalance was executed. Easier to show with an example:
**T0:** There are 3 members in the system.
**T1:** Users executes the rebalance command. 
**T2:** Rebalance finishes and control is returned to the `gfsh` command while a member joins the distributed system.
**T3:** The command use `cache.getMembers().size()` in the output , which at this time is 4, not 3; but the rebalance was executed only on the original 3 members and not the one that just joined.

Makes sense?.

Reply:
Yes, you are right. I add some sleep in between rebalance finished and output is not yet shown and it has a wrong result. 
I think that change is needed in RebalanceRegionResult, add new functions to get members.
I will close this PR and go on with different implementation.
Thanks @jujoramos !